src/fragments/lib/auth/ios/signin/60_auth_flow_types.mdx

```diff
@@ -0,0 +1,16 @@
+## Auth flow types
+
+`AWSCognitoAuthPlugin` allows you to switch between different auth flows while initiating signIn. You can configure the flow in the `amplifyconfiguration.json` file or pass the `authFlowType` as a runtime parameter to the `signIn` api call.
```

Comment:
1. Does customer configure all supported flows in `amplifyconfiguration.json`?
2. Does customer define a default flow in config, or must they always supply the flow at runtime?

Reply:
@royjit - Wanted to bump re: @palpatim 's questions. Thanks!

Reply:
Closing since this info is covered in another PR - https://github.com/aws-amplify/docs/pull/4768/files